package log_server;

message LogRequest
{
    optional bytes log_content = 1; // log
    optional bytes echo = 2; // 回显字段
    optional bytes extra = 3; // 回显字段
}

message LogResponse
{
    required int32 retcode = 1; // 返回码
    optional bytes retmsg = 2; // 返回信息
    optional bytes echo = 3; // 回显字段
    optional bytes extra = 4; // 其他信息
}

